index.html

```diff
@@ -34,3 +34,223 @@ <h1 class="homepage-heading">Emmanuel <br>
+                    <div class="union">
+                    <img class="arrow" src="images/Union.png" alt="union">
+                    </div>
+                    <div class="see-project">
+                        <p class="see"><a href="#">see this project</a></p>
+                    </div>
+                </div>
+            </div>
+            <div class="projects-project6"></div>
+            <div class="projects-img2">
+                <p class="heading-blue-background">Shell Descripts Project</p>
+                <div class="buttons">
+                    <button class="button1">HTML/CSS</button>
+                    <button class="button2">Ruby on Rails</button>
+                    <button class="button3">JavaScript</button>
+                </div>
+                <div class=rectangle-and-arrow>
+                    <img class="rectangle" src="images/Rectangle20.png" alt="Rectangle">
+                    <div class="union">
+                    <img class="arrow" src="images/Union.png" alt="union">
+                    </div>
+                    <div class="see-project">
+                        <p class="see"><a href="#">see this project</a></p>
+                    </div>
+                </div>
+            </div>
+            <section class="About">
+                <div class="about-container">
+                    <div class="about-heading">
+                        <p class="headingg">About</p>
+                    </div>
+                    <div class="about-me">
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+                    </div>
+                        <div class=rectangle-and-arrow2>
+                            <button class="about-btn"><a href="#"> Get my Cv</a></button>
+                            <div class="union">
+                            <img class="arrow2" src="images/Union (1).png" alt="union">
+
+                            </div>
+                            <div class="rectanlge45">
+                                <img src="images/Rectangle 45.png" alt="" class="rectangle3">
+
+                            </div>
+                            <div class="rec41">
+                                
+                                <p class="rec41text">Languages</p>
+                                <ul>
+                                    <li>JavaScript(ES%/ES6)</li>
+                                    <li>Ruby</li>
+                                    <li>HTML</li>
+                                    <li>CSS</li>
+                                </ul>
+                                <div class="number">
+                                    <img src="images/number.png" alt="">
+                                </div>
```

Comment:
- [ ] The screen reader will interact with these images as content and deliver it to the user, while it is for decorative purposes only,  so for semantic reasons I highly suggest setting it as a background image instead 💯  

For more information feel free to check [HTML img tag vs CSS background-image](https://maheshkonne.medium.com/html-img-tag-vs-css-background-image-23f9e2e9c8aahttps://maheshkonne.medium.com/html-img-tag-vs-css-background-image-23f9e2e9c8aa)